source/quickstart.txt

```diff
@@ -37,6 +37,10 @@ the rest of the {+c2c-product-name+} documentation.
   meet the minimum patch :ref:`version requirements
   <c2c-server-version-compatibility>`.
 
+- The source cluster may remain active during sync. After it
+  performs the initial sync, ``mongosync`` continues to sync new
```

Comment:
The second sentence here could imply that it doesn't sync during initial sync. I think you can just say that it can remain active during the sync because mongosync syncs new writes on the source cluster for the lifetime of the migration

Reply:
Fixed